rewrite-maven/src/main/java/org/openrewrite/maven/ChangeManagedDependencyGroupIdAndArtifactId.java

```diff
@@ -100,6 +103,17 @@ public Validated<Object> validate() {
         if (newVersion != null) {
             validated = validated.and(Semver.validate(newVersion, versionPattern));
         }
+        validated =
```

Comment:
As compared to the non-managed variant this one lacks the check that there's a `newGroupId` or `newArtifactId`. Was that a conscious choice, or would that make sense to add here too?
```suggestion
        validated = validated.and(required("newGroupId", newGroupId).or(required("newArtifactId", newArtifactId)));
        validated =
```

Reply:
Nevermind, was reading top to bottom and the tests do indicate this is checked.